content/en/blog/2025/stability-proposal-announcement.md

```diff
@@ -0,0 +1,303 @@
+should prioritize instrumentations that align to existing conventions and
+deprioritize others.
+
+Second, we'd like to make it easier for maintainers to ship stable
+instrumentations. If an instrumentation's API surface is stable, then we believe
+that semantic convention stability should not block the stabilization of that
+instrumentation library. This means that we'll need to be thoughtful in
+providing migration pathways for telemetry as operators upgrade to new major
+versions of instrumentation libraries.
+
+Finally, we'd like to make it easier for third-parties to publish their own
+semantic conventions by formalizing and stabilizing necessary parts of the
+semantic conventions in order for other organizations to ship conventions for
+their libraries, frameworks, tech stacks, etc.
+
+In order to accomplish this, we're looking for feedback on several areas from
+maintainers and end-users -- especially around the maturity/lifecycle of
+semantic conventions, as well as what's missing in terms of federating semantic
+conventions. We are more flexible on proposals here, but our outcomes aren't.
+Remember, a core goal of the project is to encourage other libraries, tools, and
+frameworks to
+[natively adopt OpenTelemetry](https://www.youtube.com/watch?v=l8xiNOCIdLY) --
+semantic conventions are a big part of that.
+
+## 3. Confident and Stable Releases.
+
+## Background
+
+OpenTelemetry isn't just a single binary deployed into a Kubernetes cluster.
+Subtle differences in everything from configuration to telemetry output between
+different versions of instrumentation libraries, collector receivers, and SDKs
+can cause a real headache for adopters. In addition, the rapid release cadence
+of many components causes real difficulty for end users, especially around the
+Collector. Enterprise deployments and upgrades are slow, deliberate things --
+teams simply do not have the bandwidth to validate and roll out changes at the
+cadence we ship.
+
+## Goals and Objectives
+
+Ultimately, our goal here is to make it easier for large organizations to deploy
+OpenTelemetry. Please keep in mind that in many organizations, 'deployment' and
+'upgrades' are non-trivial tasks that involve many teams and stakeholders across
+different business units or areas of responsibility including security.
+
+Our current proposal is the creation of a Release SIG that will be responsible
+for creating a schedule of 'epoch' releases for OpenTelemetry. These epoch
+versions would essentially be a manifest pointing to a tested, documented, and
+stable set of components that meet project stability requirements.
+
+This is not a trivial undertaking, to be clear. It is also not as well-defined
+as the earlier proposals in this blog -- those efforts will communicate many of
+the requirements these epoch releases must follow, after all. To our maintainers
+and contributors, though, a _non-goal_ of this effort is to force you to change
+your versioning or release cadence.
```

Comment:
maybe reword to remove "non-goal", it read to me as a "non-goal but outcome of this effort ..."